grafana/risingwave-user-dashboard.dashboard.py

```diff
@@ -541,7 +541,7 @@ def section_network(outer_panels):
                     [
                         panels.target(
                             f"{metric('batch_exchange_recv_row_number')}",
-                            "{{query_id}} : {{source_stage_id}}.{{source_task_id}} -> {{target_stage_id}}.{{target_task_id}}",
```

Comment:
`NODE_LABEL` is needed here, unless you merge them in the PromQL query.

Reply:
Oh, the problem also applies to the "Batch" panel of dev dashboard. Please help to fix by the way.